bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gems/CanopusWfs.java

```diff
@@ -117,6 +127,10 @@ public PatrolField getPatrolField() {
         return ctx.getAOComponent().filter(ado -> ado instanceof Gems).map(a -> patrolField);
     }
 
+    public static Area getGuideStarWindow() {
+        return GS_WINDOW;
```

Comment:
I guess we should probably `clone()` this since it's mutable.

Reply:
👍